docs/index.md

```diff
@@ -28,11 +28,11 @@ resource "logdna_view" "http500" {
 
 ## Pre-requirements and considerations
 Before using Terraform for creating resources in LogDNA, review the following notes:
-- Verify Terraform is [installed](https://learn.hashicorp.com/tutorials/terraform/install-cli). The minimum supported version is 0.12.0 and can be checked by running `terraform version`
-- Have the service key for your Organization available. To obtain the service key for your LogDNA Organization, go to the LogDNA dashboard and navigate to **Settings > Organization > API Keys** or follow this link [here](https://app.logdna.com/manage/api-keys)
-- Be aware that the underlying LogDNA Configuration API has a rate limit of 50 requests at a time; therefore, when using the LogDNA Terraform provider, there is also a limit of 50 resource operations at a time
-- If you do not provide a specific base URL in the provider configuration, the base url defaults to `https://api.logdna.com`
-- Authentication is handled via the `servicekey` parameter and can be set in the provider configuration
+- Verify Terraform is [installed](https://learn.hashicorp.com/tutorials/terraform/install-cli). The minimum supported version is 0.12.0 and can be checked by running `terraform version`.
+- Have the service key for your Organization available. To obtain the service key for your LogDNA Organization, go to the LogDNA dashboard and navigate to **Settings > Organization > API Keys** or follow this link [here](https://app.logdna.com/manage/api-keys).
+- Be aware that the underlying LogDNA Configuration API has a rate limit of 50 requests at a time; therefore, when using the LogDNA Terraform provider, there is also a limit of 50 resource operations at a time.
+- If you do not provide a specific base URL in the provider configuration, the base url defaults to `https://api.logdna.com`.
+- Authentication is handled via the `servicekey` parameter and can be set in the provider configuration.
 
```

Comment:
Please add this bullet item to the end of the list:

- Note that if you create a new View using Terraform, but then delete the View by using the LogDNA Dashboard UI and *not* Terraform, then Terraform will not be aware of the deletion and displays an `Error: Resource Not Found` message. Should this occur, the work around is to use `terraform state rm` on the deleted View resource.

Reply:
Also,I am really not sure that the above is enough info... is that all we can tell them, that if they mess it up and get that error, then to use the `terraform state rm` command?

Reply:
yup so the user will be unable to update the resource and by running terraform state rm they'll be able to get the two in sync.